### ISSUE TITLE
Content-based detection of PDFs fails in Chrome 52 (Material Design PDF viewer)

### ISSUE THREAD

**Issue by robertknight** [hypothesis/h#3293]
### Steps to reproduce
1. Install Chrome 52
2. Visit http://resource-cms.springer.com/springer-cms/rest/v1/content/7368240/data/v1/Concatenated+PDF+12S1
3. Activate Hypothesis extension

### Expected behaviour
PDF should be opened in the bundled PDF.js viewer and Hypothesis sidebar should appear

### Actual behaviour
Hypothesis icon shows as active but PDF.js viewer does not load

### Browser/system information
Chrome 52 with material design-based PDF viewer. Note the references to the Polymer library in the PDF viewer's HTML content.

### Additional details
The Hypothesis plugin detects PDFs by looking for an `<embed>` tag using this selector: `embed[type="application/pdf"][name="plugin"]`. This does not match the tag in Chrome 52:

```html
<embed id="plugin" type="application/x-google-chrome-pdf" src="http://resource-cms.springer.com/springer-cms/rest/v1/content/7368240/data/v1/Concatenated+PDF+12S1" stream-url="blob:chrome-extension://mhjfbmdgcfjbbpaeojofohoefgiehjai/F29F38E5-18FB-4754-A8A0-9C50C18F83BE" headers="Accept-Ranges: bytes
Age: 17623
Cache-Control: max-age=31536000, public
Connection: keep-alive
Content-Encoding: gzip
Content-Language: en-US
Content-Length: 630703
Content-Type: application/pdf;charset=UTF-8
Date: Sun, 08 May 2016 16:09:00 GMT
ETag: 4a2b8844e6d340f081cfd87c68919f95
Vary: Accept-Encoding
Via: 1.1 varnish-v4
X-Cacheable: YES
X-Varnish: 146974730 145348381
" background-color="0xFF525659" top-toolbar-height="56" full-frame="">
```

**Comment by dwhly** [comment]
How difficult is this to fix?  Important issue.

**Comment by robertknight** [comment]
I'm mistaken. I'm not sure what I did yesterday that caused activating H on a PDF viewer to fail but was working when I tested again this morning and the description is mistaken.

The confusion was caused by the Chrome PDF viewer consisting of a native PDF viewer embedded in an HTML document providing the PDF viewer controls which in turn is instantiated by an `<embed>` tag in the top-level document. ie. Like this:

```html
<html>
<body>
  <embed name="plugin" type="application/pdf" ...>
    <!-- PDF viewer UI !-->
    <embed id="plugin" type="application/x-google-chrome-pdf"></embed>
  </embed>
</body>
</html>
```

If you open the dev tools using Ctrl+Shift+I on a PDF viewer tab you get the outer document. If you right-click anywhere in the page and open the dev tools via the 'Inspect' option, you get the inner document (ie. the PDF viewer UI). When I wrote this bug report last night I was looking at the inner document. When the Hypothesis extension detects the content type however it looks at the outer document.
